package.json

```diff
@@ -78,44 +78,44 @@
     "@types/react-redux": "7.1.20",
     "@types/react-router-dom": "^5.3.3",
     "@types/webpack": "^5.28.0",
-    "@typescript-eslint/eslint-plugin": "^5.35.1",
-    "@typescript-eslint/parser": "^5.35.1",
+    "@typescript-eslint/eslint-plugin": "^5.37.0",
+    "@typescript-eslint/parser": "^5.37.0",
     "@xstate/test": "^0.5.1",
     "@yarnpkg/lockfile": "^1.1.0",
     "aphrodite": "^2.4.0",
     "commander": "^9.4.0",
     "copy-webpack-plugin": "^11.0.0",
     "css-loader": "^6.7.1",
-    "eslint": "8.22.0",
-    "eslint-plugin-formatjs": "^4.3.0",
+    "eslint": "8.23.1",
+    "eslint-plugin-formatjs": "^4.3.1",
     "eslint-plugin-jest-dom": "^4.0.2",
     "eslint-plugin-jsdoc": "^39.3.6",
     "eslint-plugin-markdown": "^3.0.0",
     "eslint-plugin-patternfly-react": "^4.78.0",
     "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-react": "^7.31.2",
-    "eslint-plugin-simple-import-sort": "^7.0.0",
+    "eslint-plugin-react": "^7.31.8",
+    "eslint-plugin-simple-import-sort": "^8.0.0",
     "eslint-plugin-sort-keys-fix": "^1.1.2",
-    "eslint-plugin-testing-library": "^5.6.0",
+    "eslint-plugin-testing-library": "^5.6.4",
     "git-revision-webpack-plugin": "^5.0.0",
     "glob": "^8.0.3",
     "html-loader": "^4.1.0",
     "html-replace-webpack-plugin": "^2.6.0",
     "html-webpack-plugin": "^5.5.0",
-    "inquirer": "^8.2.4",
+    "inquirer": "^9.1.1",
```

Comment:
Can't update this, it doesn't support `require`, so it breaks the startup script